src/main/kotlin/racingcar/validator/InputArgumentValidator.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar.validator
+
+interface InputArgumentValidator {
```

Comment:
메서드가 하나뿐인 인터페이스라면 자바의 FunctionalInterface 애노테이션처럼
인터페이스에 fun을 붙혀줄 수 있습니다. 
```suggestion
fun interface InputArgumentValidator {
```